page.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Noisemaker</title>
+	<script src="musicFunction.js"></script>
+	<script src="Tone.js"></script>
+	<style>
+	body  {
+	  background-color: #85F697;
+	   margin: 0;
+	}
+	h1{
+	font-family: "Comic Sans MS", cursive, sans-serif;
+	}
+	.flower {
+		height: 50px;
+		width: 50px;
+		cursor: pointer;
+		position: absolute;
+		animation: animate 10s linear 0s infinite alternate;
+	}
+	@keyframes animate {
+		0%	 {left: 500px; top: 300px;}
+		25%  {left: 200px; top: 590px;}
+		50%  {left: 200px; top: 200px;}
+		75%  {left: 566px; top: 200px;}
+		100% {left: 200px; top: 485px;}
+	}
+	div.container {
+		width: 50px;
+		height:50px;
+		position:fixed;
+	}
+	</style>
```

Comment:
these are quite a few lines for styling. might be better to include a .css file with this information in stead.

Reply:
Yes if I were to develop this further I would definitely separate style from structure :)